databases/package.json

```diff
@@ -43,25 +44,21 @@
     "eslint": "^7.19.0",
     "eslint-plugin-import": "^2.22.1",
     "glob": "^7.1.6",
-    "gulp": "^3.9.1",
```

Comment:
Many packages were not in use, and are needed only for extensions, not NPM packages.